test/linkify-matrix-test.ts

```diff
@@ -0,0 +1,227 @@
+import * as linkify from "linkifyjs";
+import linkifyMatrix from '../src/linkify-matrix';
+
+describe('linkify-matrix', () => {
+    beforeAll(() => {
+        linkifyMatrix(linkify);
```

Comment:
normally we do this right after the import, we should probably do that to match all other callers